internal/logging/context.go

```diff
@@ -0,0 +1,14 @@
+package logging
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-log/tfsdklog"
+)
+
+// InitContext creates SDK logger contexts.
+func InitContext(ctx context.Context) context.Context {
+	ctx = tfsdklog.NewSubsystem(ctx, SubsystemHelperSchema, tfsdklog.WithLevelFromEnv(EnvTfLogSdkHelperSchema))
+
+	return ctx
+}
```

Comment:
```suggestion
func WithLogging(ctx context.Context) context.Context {
	return tfsdklog.NewSubsystem(ctx, SubsystemHelperSchema, tfsdklog.WithLevelFromEnv(EnvTfLogSdkHelperSchema))
}
```

Reply:
or `WithLogger`. Just to kinda mimic stuff like https://pkg.go.dev/context#WithValue